example/client.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	gpb "github.com/BetterGR/homework-microservice/homework_protos" // Replace with your package path
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	addr = flag.String("addr", "localhost:1234", "the address to connect to") // Set port to 1234

```

Comment:
Lets avoid using global variables. Instead, the address can be saved in a const, and this variable can be built when needed.